modules/KalturaSupport/resources/mw.KBaseMediaList.js

```diff
@@ -426,7 +426,7 @@
 			var thumbUrl = kWidgetSupport.getKalturaThumbnailUrl(
 				$.extend( {}, this.baseThumbSettings, {
                     'url': this.embedPlayer.evaluate('{mediaProxy.entry.thumbnailUrl}'),
-					'vid_sec': parseInt( time / 1000 )
```

Comment:
@lianbenjamin did you check why the parsing was added?

Reply:
@yairans yes, tried to search for a reason using git blame but didn't find anything specific to parseInt. maybe BE in the past didn't support float number...